### PR TITLE
zephyr: add L2CAP ECFC tests

### PR DESCRIPTION
**Pull request by KKopyscinski**
Added tests that became supported by Zephyr. Modified/added
wids to allow these tests to pass. Wids 106-108 do not require
any action, since PTS verifies received responses on its own.

### CODE REVIEW

**Comment by KKopyscinski** [comment]
These tests require following Zephyr patches:
https://github.com/zephyrproject-rtos/zephyr/pull/34266
https://github.com/zephyrproject-rtos/zephyr/pull/33979
https://github.com/KKopyscinski/zephyr/commit/da3864fe2d88279ba87691ae27b296a562534b41

Tests that aren't passing currently:
L2CAP/ECFC/BV-15-C - requires patch in Zephyr, to check keysize on existing connection on L2CAP Connection Request
L2CAP/ECFC/BV-22-C - requires adding function to bttester app (reconfigure request)
L2CAP/ECFC/BI-01-C - this test requires at least 2 initial credits on new L2CAP connection, but in Zephyr there's always one, see https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/bluetooth/host/l2cap.c#L807
L2CAP/ECFC/BI-02-C - possible PTS issue, spec says that IUT should send credit indication (which it does) but PTS is waiting for credit request

**Comment by michal-narajowski** [comment]
`L2CAP/ECFC/BI-01-C - this test requires at least 2 initial credits on new L2CAP connection, but in Zephyr there's always one` - We should create an issue for this one in Zephyr repo.

`L2CAP/ECFC/BV-15-C - requires patch in Zephyr, to check keysize on existing connection on L2CAP Connection Request
L2CAP/ECFC/BV-22-C - requires adding function to bttester app (reconfigure request)` - You can submit these patches now, before merging this.

**Comment by KKopyscinski** [comment]
There's also issue with L2CAP/ECFC/BV-13-C - Insufficient authorization shall be returned by l2cap server on connection, so bttester app should also be modified to return such error depending on some condition. In real world this should be done by asking user to confirm accepting connection, but here we can just introduce some config variable

File: wid/l2cap.py [hunk 1]
```diff
@@ -390,7 +373,7 @@ def hdl_wid_111(desc):
     l2cap = stack.l2cap
 
     channels = l2cap.rx_data_get_all(10)
-    if not (len(channels) == 1):
+    if not (len(channels) >= 1):
```
Comment (michal-narajowski) [inline wid/l2cap.py:376]: so `if len(channels) == 0`?

Comment (KKopyscinski) [reply wid/l2cap.py:376]: Yes :)

**Review by michal-narajowski — CHANGES_REQUESTED** [review]

**Review by michal-narajowski — APPROVED** [review]

**Pull request merged**